### DISCUSSION TITLE
Video embedding creates large white space

### DISCUSSION THREAD

**Discussion by nantonel** [mmistakes/so-simple-theme#381] [Q&A] [answered]
Hi! Thank you for this theme, it's really nice! 😄 

I'm having some troubles with video embedding. As you can see [here](https://nantonel.github.io/2018/08/09/Juliacon-StructuredOptimization.html) when I embed a youtube video  a white space is created. 

The source code for that page is [here](https://github.com/nantonel/nantonel.github.io/blob/master/_posts/2018-08-09-Juliacon-StructuredOptimization.md) and is really similar to the one reported on the [so-simple examples](https://github.com/mmistakes/so-simple-theme/blob/master/example/_posts/2010-06-02-post-video-youtube.md).
The effect is also present in the [list of blog](https://nantonel.github.io/blog/) (you have to scroll down a bit to see that).

I'm not sure what causes, perhaps some of my settings?
I thank you for your help.

**Comment by mmistakes** [comment, marked as answer]
Your issue is you're using [FitVids](https://github.com/nantonel/nantonel.github.io/blob/master/assets/js/plugins/jquery.fitvids.js) along with the responsive embed. You don't need to do both.
Since you have FitVids installed you can just embed the YouTube videos without using the `responsive-embed` include.

[That include has some CSS](https://github.com/mmistakes/so-simple-theme/blob/master/_sass/so-simple/utilities/_responsive-embed.scss) that adds padding to make the video responsive. FitVids does the same so you're doubling up the padding.

**Comment by nantonel** [comment]
Thank you for your quick response. Removing those scripts fixed it.
